### DISCUSSION TITLE
Unpacking CosmoSkyMed Second Generation data

### DISCUSSION THREAD

**Discussion by svenborgstrom** [isce-framework/isce2#637] [Q&A]
Good mornig everybody,
I tried to unpack a CosmoSkyMed second generation data (what ASI calls CSG) with the "unpackFrame_CSK.py" script but it doesn't work (see the attached file).
As far as I can understand, ISCE is supporting only Cosmo first generation (CSK) data, is it right?
If so, is there someone who will take care in the next future to set a new script, let me say "unpackFrame_CSG.py" for this?
Cheers,
Sven.

<img width="1349" alt="Unpack_Cosmo2ndGeneration" src="https://user-images.githubusercontent.com/93733824/206925010-bbc4de40-27ba-482f-879b-b7a25c7faf30.png">

**Comment by SoleHeredia** [comment]
Hi, I couldn´t do the dense_offset with CSG, so I have the same doubt, does ISCE support CSG files? Did you find any answer?
Thank you!

**Reply by svenborgstrom** [reply]
Hi,
I asked some colleagues and I was told that unfortunately at this moment ISCE doesn't support Cosmo Second Generation data processing.
Cheers,
Sven.

**Reply by SoleHeredia** [reply]
Thank you!
Cheers, 
Sole
